### ISSUE TITLE
Make selected tab more apparent

### ISSUE THREAD

**Issue by drewvolz** [StoDevX/AAO-React-Native#2742]
Either add a color highlight to it, make it darker, or something. I can hardly tell on most views which tab is active. 

The problem becomes even worse with Night Shift.

**Comment by drewvolz** [comment]
I will revisit this if it truly becomes a problem.

**Comment by drewvolz** [comment]
@hawkrives and I compared the blacks and grays of the icons in the tab bars for iBooks vs AAO. We still think this is an "indistinguishable-in-low-light" problem. 

App | State | Color
--|--|--
St Olaf | active | rgb(26,25,26)
iBooks | active | rgb(0,0,0)
St Olaf | inactive | rgb(128,128,128)
iBooks | inactive | rgb(144,144,144)

St. Olaf | iBooks
--|--
![photo_2018-08-14_11-15-13](https://user-images.githubusercontent.com/5240843/44107016-5ae45a74-9fb3-11e8-8fb7-4e3012fac936.jpg) | ![photo_2018-08-14_11-15-07](https://user-images.githubusercontent.com/5240843/44107018-5b05aa76-9fb3-11e8-8dad-964d6e041ab3.jpg)

**Comment by hawkrives** [comment]
I'm planning to put make the inactive color configurable, and then change the colors on iOS to match iBooks.

**Comment by hawkrives** [comment]
Closed by #2929, I hope.
